lang-model/src/main/java/module-info.java

```diff
@@ -0,0 +1,5 @@
+module jakarta.cdi.model {
```

Comment:
I'd probably call it `jakarta.cdi.lang.model`.

Reply:
Ok, updated.